CadRevealComposer/ModelMetadata.cs

```diff
@@ -32,8 +32,7 @@ public static string Serialize(ModelMetadata metadata)
     // returns false otherwise
     public bool CheckValue(string key, string value)
     {
-        string? realValue;
-        if (_metadata.TryGetValue(key, out realValue))
+        if (_metadata.TryGetValue(key, out var realValue))
         {
             return realValue!.Equals(value, System.StringComparison.OrdinalIgnoreCase);
```

Comment:
How come we need to expect realValue to be non-null here? ref `!` here `realValue!.Equals`